scripts/repair_missing_webp_images.py

```
@@ -0,0 +1,201 @@
+        "prod_success": False,
+        "error": None,
+    }
+
+    try:
+        # Download JPEG from dev bucket (should exist in both, but dev is source of truth)
+        logger.info(f"  Downloading JPEG from s3://{dev_bucket}/{jpeg_key}")
+        image = download_jpeg_from_s3(s3_client, dev_bucket, jpeg_key)
+        logger.info(f"  Image size: {image.width}x{image.height}")
+
+        if dry_run:
+            logger.info(f"  [DRY RUN] Would upload WebP to both buckets")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)